docs/fundamentals/code-analysis/style-rules/ide0130.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "IDE0130: Namespace does not match folder structure"
+description: "Learn about code analysis rule IDE0130: Namespace does not match folder structure"
+ms.date: 12/05/2022
+ms.topic: reference
+f1_keywords:
+- IDE0130
+helpviewer_keywords:
+- IDE0130
```

Comment:
@gewarren Do we add the code-style option in f1_keywords and helpviewer_keywords?

Reply:
It's probably a good idea.